src/Paket.Core/ConfigFile.fs

```diff
@@ -88,14 +88,17 @@ let private readPassword (message : string) : string =
     password
 
 let getAuthFromNode (node : XmlNode) = 
-    let username = node.Attributes.["username"].Value
-    let password = node.Attributes.["password"].Value
-    let salt = node.Attributes.["salt"].Value
-
-    username, Decrypt salt password
-
-let private createSourceNode (credentialsNode : XmlNode) source =
-    let node = credentialsNode.OwnerDocument.CreateElement("credential")
+    match node.Name.ToLowerInvariant() with
+    | "credential" ->
```

Comment:
No I guess it is. But I don't know that part of the code very well so I was
just asking.
On Oct 4, 2015 23:28, "Jon Canning" <notifications@github.com> wrote:

> In src/Paket.Core/ConfigFile.fs
> <https://github.com/fsprojects/Paket/pull/1111#discussion_r41102639>:
>
> > @@ -88,14 +88,17 @@ let private readPassword (message : string) : string =
> >      password
> >
> >  let getAuthFromNode (node : XmlNode) =
> > -    let username = node.Attributes.["username"].Value
> > -    let password = node.Attributes.["password"].Value
> > -    let salt = node.Attributes.["salt"].Value
> > -
> > -    username, Decrypt salt password
> > -
> > -let private createSourceNode (credentialsNode : XmlNode) source =
> > -    let node = credentialsNode.OwnerDocument.CreateElement("credential")
> > +    match node.Name.ToLowerInvariant() with
> > +    | "credential" ->
>
> It is for me but you've got me worried now. Not sure how else to test it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fsprojects/Paket/pull/1111/files#r41102639>.
>